dotnet-client-libraries/IntroductionToWolverineLibrary/IntroductionToWolverineLibrary/Handlers/HandlerMethods.cs

```diff
@@ -0,0 +1,21 @@
+﻿using IntroductionToWolverineLibrary.Models;
+
+namespace IntroductionToWolverineLibrary.Handlers;
+
+public class OrderHandler
+{
+    public static void Handle(NewBookOrder order)
```

Comment:
This function can also be written in lamda expression

Reply:
Changed it to lambda expression.